src/main/webapp/resources/ProjectPage.html

```diff
@@ -0,0 +1,352 @@
+                            <div class="progress">
+                                <div class="progress-bar progress-bar-info" style="width: 20%"></div>
+                            </div>
+                            <h4 class="text-right">Project need $80000</h4>
+                            <!--todo list of investments for project creator-->
+                        </div>
+                        <div class="panel-footer">
+                            <button type="button" class="btn btn-success btn-block" data-toggle="modal"
+                                    data-target="#modal-invest">I
+                                want to invest
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <!--------------------------------------------->
+        </div>
+        <div class="col-md-3 col-md-offset-8 col-lg-offset-9 col-lg-2 hidden-sm hidden-xs " style="position: fixed">
+            <div class="item">
+
+                <!-- PRICE ITEM -->
+                <!--The same situation with classes. I mean panel-red to panel-danger-->
+                <div class="panel price panel-danger">
+                    <div class="panel-heading">
+                        <h4><strong>Min investment</strong></h4>
+                        <h3 class="text-right">$10000</h3>
+                    </div>
+                    <div class="panel-body text-right">
+
+                        <h5>Total project investment $100000</h5>
+                        <div class="progress">
+                            <div class="progress-bar progress-bar-info" style="width: 20%"></div>
+                        </div>
+                        <h5>Project need $80000</h5>
+                        <!--todo list of investments for project creator-->
+                    </div>
+                    <div class="panel-footer">
+                        <button type="button" class="btn btn-success btn-block" data-toggle="modal"
+                                data-target="#modal-invest">I
+                            want to invest
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+
+        <div class="footer">
+            <div class="container">
+                <div class="row centered">
+                    <a href="#"><span class="fa fa-twitter"></span></a>
+                    <a href="#"><span class="fa fa-facebook"></span></a>
+                    <a href="#"><span class="fa fa-vk"></span></a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+</div>
+
+<!--Modal-->
+<div class="modal fade" id="modal-invest">
+    <div class="modal-dialog">
+        <div class="modal-content ">
+            <div class="modal-header">
+                <button class="close" type="button" data-dismiss="modal">
+                    <i class="fa fa-close"></i>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)